docs/index.html

```diff
@@ -10,72 +10,68 @@
     <title>CtTableFor: Rails Responsive Table. table_for helper</title>
   </head>
 <body>
+<div class="container">
 <div class="alert alert-info">
-  <p>Has d'incloure al <code>Gemfile</code>:<br>
+  <p>Needed to include to <code>Gemfile</code>:<br>
   <code>gem 'ct_table_for', '~> 0.1.15.beta'</code>
   </p>
 </div>
 <div class="box">
   <div class="box-header">
-    <h3 class="box-title">Utilitzant un ActiveRecord</h3>
+    <h3 class="box-title">Using an ActiveRecord</h3>
   </div>
   <div class="box-body ">
-    <p> Els dos camps requerits son: <strong>Model</strong> i una <strong>col·lecció d'ActiveRecord</strong>, automàticament renderitzarà tots els attributs de l'activerecord "</p>
-    <p> Per exemple de la següent manera <code>&#x3C;%= table_for User, @users %&#x3E;</code></p>
-    <%= table_for User, @users %>
+    <p> The two required fields are: a <strong>Model</strong> and a <strong>collection of ActiveRecord</strong>, automatically will render all the attributes of the activerecord "</p>
+    <p> For example by the following way <code>&#x3C;%= table_for User, @users %&#x3E;</code></p>
   </div>
 </div>
 
 <div class="box">
   <div class="box-header">
-    <h3 class="box-title">Utilizant uns camps concrets del Model</h3>
+    <h3 class="box-title">Using concrete fields of the Model</h3>
```

Comment:
concrete = formigó.

You can say something like "Using *specific* fields of the Model"